javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,51 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+// my codes goes here
+const productContainer = document.getElementById("productDiv");
+const searchProduct = document.getElementById("searchProduct");
+const maxPrice = document.getElementById("maxPrice");
+const productUl = document.createElement("ul");
+productContainer.appendChild(productUl);
+
+function displaySearchedProduct(searchedProducts) {
+  while (productUl.children.length != 0) {
+    productUl.removeChild(productUl.childNodes[0]);
+  }
+  searchedProducts.forEach((obj) => {
+    const titleLi = document.createElement("li");
+    const titleH1 = document.createElement("h1");
+    titleH1.innerHTML = obj.name;
```

Comment:
Generally, `h1` should only be used for the title of the whole page. Though you don't necessarily have to restrict yourself to this 'rule', this is just a list item. You can just modify its text with innerText.